streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/processor/StreamProcessor.java

```diff
@@ -17,17 +23,26 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(StreamProcessor.class);
 
   protected List<Collector> collectors;
-  protected RuntimeContext runtimeContext;
+  protected StreamRuntimeContext runtimeContext;
   protected P operator;
+  private KeyStateBackend keyStateBackend;
 
   public StreamProcessor(P operator) {
     this.operator = operator;
   }
 
   @Override
-  public void open(List<Collector> collectors, RuntimeContext runtimeContext) {
+  public void open(List<Collector> collectors, StreamRuntimeContext runtimeContext) {
     this.collectors = collectors;
     this.runtimeContext = runtimeContext;
+    int maxPara = Integer.valueOf(runtimeContext.getConfig().getOrDefault(
```

Comment:
```suggestion
    int maxParallelism = Integer.valueOf(runtimeContext.getConfig().getOrDefault(
```

`Para` can be `Parameter`, we don't have to use abbr. .

Reply:
ok